common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisPilotingManager.java

```diff
@@ -259,7 +259,7 @@ private void checkThrottleStatesForFlight() {
             this.endFlightEarly(false);
         }
 
-        if (isInFlight && this.canEndFlight()  && !this.isLanding() && !this.isTakingOff() && (this.isHandbrakeOn || this.throttleStage == 0)) {
+        if (isInFlight && this.canEndFlight() && !this.isLanding() && !this.isTakingOff() && (this.isHandbrakeOn || this.throttleStage == 0)) {
```

Comment:
Not sure if you fixed it, but the throttleStage being 0 may cause the Flight ended notification noise to play

Reply:
Not sure what this is referring to. Could you attach a video?

Reply:
I can’t at the moment but we play a sound when the flight is finished that alerts the user - I remember we had issues with it playing when it shouldn’t